boards/DIM/Src/App/states/App_DriveState.c

```diff
@@ -60,33 +160,25 @@ static void DriveStateRunOnTick100Hz(struct StateMachine *const state_machine)
     const bool aux_switch_on   = App_BinarySwitch_IsTurnedOn(aux_switch);
 
     App_CanTx_DIM_Switches_StartSwitch_Set(start_switch_on ? SWITCH_ON : SWITCH_OFF);
-    App_CanTx_DIM_Switches_AuxSwitch_Set(aux_switch ? SWITCH_ON : SWITCH_OFF);
+    App_CanTx_DIM_Switches_AuxSwitch_Set(aux_switch_on ? SWITCH_ON : SWITCH_OFF);
 
     struct RgbLed *board_status_leds[NUM_BOARD_LEDS] = { [BMS_LED] = App_DimWorld_GetBmsStatusLed(world),
                                                          [DCM_LED] = App_DimWorld_GetDcmStatusLed(world),
                                                          [DIM_LED] = App_DimWorld_GetDimStatusLed(world),
                                                          [FSM_LED] = App_DimWorld_GetFsmStatusLed(world),
                                                          [PDM_LED] = App_DimWorld_GetPdmStatusLed(world) };
 
-    // TODO: Update these LEDs to reflect board status, now that error table is gone
     for (size_t i = 0; i < NUM_BOARD_LEDS; i++)
     {
         struct RgbLed *board_status_led = board_status_leds[i];
 
-        if (i == BMS_LED)
+        if ((*has_fault_funcs[i])())
         {
```

Comment:
This line can just be `if (has_fault_funcs[i]())`

Reply:
I believe this is because you can just call function ptrs directly, you don't need to deref them first